Client.xcodeproj/project.pbxproj

```diff
@@ -4666,6 +4666,7 @@
 		282731771ABC9BE800AA1954 /* Fennec */ = {
 			isa = XCBuildConfiguration;
 			buildSettings = {
+				ALWAYS_SEARCH_USER_PATHS = NO;
 				APPLICATION_EXTENSION_API_ONLY = YES;
 				DEFINES_MODULE = YES;
 				DYLIB_COMPATIBILITY_VERSION = 1;
```

Comment:
Nit: Since we set this to NO in every target, you can set it in the project setting instead of the each target to avoid duplication.